src/main/java/firebase/sdk/dashboard/dao/SDKDaoDatastore.java

```diff
@@ -0,0 +1,343 @@
+    Entity entity = preparedQuery.asSingleEntity();
+
+    if (entity == null) {
+      return null;
+    }
+
+    Platform platform = Platform.fromInteger(((Number) entity.getProperty("platform")).intValue());
+    String libraryName = (String) entity.getProperty("libraryName");
+    String libraryGroup = (String) entity.getProperty("libraryGroup");
+    String externalName = (String) entity.getProperty("externalName");
+    String fireEscapeName = (String) entity.getProperty("fireEscapeName");
+    String owner = (String) entity.getProperty("owner");
+    List<VersionMetadata> versionHistory = getVersionHistoryFromProperty(
+        (List<EmbeddedEntity>) entity.getProperty("versionHistory"));
+
+    SDK sdk = SDK.newBuilder()
+      .platform(platform)
+      .libraryName(libraryName)
+      .libraryGroup(libraryGroup)
+      .externalName(externalName)
+      .fireEscapeName(fireEscapeName)
+      .owner(owner)
+      .versionHistory(versionHistory)
+      .build();
+
+    return sdk;
+  }
+
+  public List<String> getSDKs(Platform givenPlatform) {
+    FilterPredicate platformPropertyFilter = makePropertyFilter("platform", givenPlatform.getValue());
+
+    Query query = new Query("SDK")
+      .setFilter(platformPropertyFilter);
+
+    PreparedQuery preparedQuery = DATASTORE.prepare(query);
+    FetchOptions fetchOptions = FetchOptions.Builder.withDefaults();
+    QueryResultIterable<Entity> results = preparedQuery.asQueryResultIterable();
+
+    if (results == null) {
+      return Arrays.asList();
+    }
+
+    ArrayList<String> sdksForPlatform = new ArrayList<>();
+    for (Entity entity: results) {
+      sdksForPlatform.add(
+          (String) entity.getProperty("libraryName"));
+    }
+
+    return sdksForPlatform;
+  }
+
+  public List<String> getSDKsEnrolledInRelease(Platform platform, String releaseName) {
+    FilterPredicate platformPropertyFilter = makePropertyFilter("platform", platform.getValue());
+    FilterPredicate releasePropertyFilter = makePropertyFilter("releaseName", releaseName);
+    CompositeFilter compositeFilter = new CompositeFilter(
+        CompositeFilterOperator.AND, Arrays.asList(
+          platformPropertyFilter,
+          releasePropertyFilter));
+
+    Query query = new Query("SDKReleaseMetadata")
+      .setFilter(compositeFilter);
+
+    PreparedQuery preparedQuery = DATASTORE.prepare(query);
+    FetchOptions fetchOptions = FetchOptions.Builder.withDefaults();
+    QueryResultIterable<Entity> results = preparedQuery.asQueryResultIterable(fetchOptions);
+
+    if (results == null) {
+      return Arrays.asList();
+    }
+
+    ArrayList<String> enrolledSDKs = new ArrayList<>();
+    for (Entity entity: results) {
+      String libraryName = (String) entity.getProperty("libraryName");
+      enrolledSDKs.add(libraryName);
+    }
+
+    return enrolledSDKs;
+  }
+
+  public SDKReleaseMetadata getSDKReleaseMetadata(Platform givenPlatform, String givenReleaseName, String givenLibraryName) {
```

Comment:
ditto. Remove 'given'. It also doesn't seem to be consistent across all the methods in this file. You would want to keep the naming consistent.